### PR TITLE
Added resources on albert model

### PR DESCRIPTION
**Pull request by JuheonChu**
# What does this PR do?

Co-author: @Adia Wu <wua@dickinson.edu>
Fixes #20055 

## Before submitting
- [o] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [o] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [o] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [o] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [o] Did you write any new necessary tests?

## Who can review?

@stevhliu @younesbelkada

### CODE REVIEW

File: docs/source/en/model_doc/albert.mdx [hunk 1]
```diff
@@ -23,14 +23,14 @@ speed of BERT:
...
-*Increasing model size when pretraining natural language representations often results in improved performance on
+_Increasing model size when pretraining natural language representations often results in improved performance on
 downstream tasks. However, at some point further model increases become harder due to GPU/TPU memory limitations,
 longer training times, and unexpected model degradation. To address these problems, we present two parameter-reduction
 techniques to lower memory consumption and increase the training speed of BERT. Comprehensive empirical evidence shows
 that our proposed methods lead to models that scale much better compared to the original BERT. We also use a
 self-supervised loss that focuses on modeling inter-sentence coherence, and show it consistently helps downstream tasks
 with multi-sentence inputs. As a result, our best model establishes new state-of-the-art results on the GLUE, RACE, and
-SQuAD benchmarks while having fewer parameters compared to BERT-large.*
+SQuAD benchmarks while having fewer parameters compared to BERT-large._
```
Comment (younesbelkada) [inline docs/source/en/model_doc/albert.mdx:33]: Could you revert these changes? 🙏

File: docs/source/en/model_doc/albert.mdx [hunk 2]
```diff
@@ -43,17 +43,62 @@ Tips:
...
+<PipelineTag pipeline="introduce" />- A blog post on [Getting Started with
+ALBERT](https://huggingface.co/albert-base-v2). - A blog post on
+[Albert-base-v2](https://huggingface.co/albert-base-v2). - A blog post on
+[albert-xlarge-v2](https://huggingface.co/albert-xlarge-v2). - A blog post on
+[Chinese
+Albert](https://huggingface.co/uer/albert-base-chinese-cluecorpussmall). - A
+blog post on [Albert-tiny](https://huggingface.co/clue/albert_chinese_tiny). -
+An article about [ALBERT: A Lite BERT for Self-supervised Learning of Language
+Representations](https://arxiv.org/abs/1909.11942)
+
```
Comment (younesbelkada) [inline docs/source/en/model_doc/albert.mdx:59]:
```suggestion
<PipelineTag pipeline="introduce" />
- A blog post on [Getting Started with ALBERT](https://huggingface.co/albert-base-v2). 
- A blog post on [Albert-base-v2](https://huggingface.co/albert-base-v2). 
- A blog post on [albert-xlarge-v2](https://huggingface.co/albert-xlarge-v2). 
- A blog post on [Chinese Albert](https://huggingface.co/uer/albert-base-chinese-cluecorpussmall).
- A blog post on [Albert-tiny](https://huggingface.co/clue/albert_chinese_tiny). 
- An article about [ALBERT: A Lite BERT for Self-supervised Learning of Language Representations](https://arxiv.org/abs/1909.11942)
```

File: src/transformers/models/distilbert/modeling_distilbert.py [hunk 3]
```diff
@@ -142,10 +141,7 @@ def __init__(self, config: PretrainedConfig):
         self.dim = config.dim
         self.dropout = nn.Dropout(p=config.attention_dropout)
 
-        # Have an even number of multi heads that divide the dimensions
-        if self.dim % self.n_heads != 0:
-            # Raise value errors for even multi-head attention nodes
-            raise ValueError(f"self.n_heads: {self.n_heads} must divide self.dim: {self.dim} evenly")
+        assert self.dim % self.n_heads == 0
```
Comment (younesbelkada) [inline src/transformers/models/distilbert/modeling_distilbert.py:144]: Could you also revert the changes here ? It seems that you have deleted this by mistake

**Review by younesbelkada — COMMENTED** [review]
Thanks so much @JuheonChu for adding the resources for ALBERT ! 
I left a couple of comments, the main ones being reverting the changes that you did by mistake on `modeling_distilbert.py`. Also make sure that the text you added is rendered correctly! Let us know if you need any help

File: docs/source/en/model_doc/albert.mdx (hunk 1 above)
Comment (JuheonChu) [reply docs/source/en/model_doc/albert.mdx:33]: Does that mean deleting "_"?

**Comment by JuheonChu** [comment]
Thank you! Will try! 
So, I deleted those undesired behaviors, and now I will look for more resources to add!

File: docs/source/en/model_doc/albert.mdx (hunk 1 above)
Comment (stevhliu) [reply docs/source/en/model_doc/albert.mdx:33]: It means you can leave the asterisks `*` instead of using an underscore `_`

File: docs/source/en/model_doc/albert.mdx [hunk 4]
```diff
@@ -67,104 +110,84 @@ This model was contributed by [lysandre](https://huggingface.co/lysandre). This
 
 ## AlbertModel
 
-[[autodoc]] AlbertModel
-    - forward
+[[autodoc]] AlbertModel - forward
```
Comment (stevhliu) [inline docs/source/en/model_doc/albert.mdx:113]: You can leave this alone as well and allow the `forward` method to be listed under the `AlbertModel` object. Same comment applies to all the other objects changed below :)

**Review by stevhliu — COMMENTED** [review]
Thanks for your contribution!

I think it might be easier to open a new PR with changes only to the `albert.mdx` file because now the `modeling_distilbert.py` has been deleted and we don't want that!

**Comment by JuheonChu** [comment]
Do you mind if I open a new Pull Request in order to contain only meaningful commits?

**Comment by stevhliu** [comment]
Yes please, that'd be great!
